src/.nuget/toolchain.ubuntu.14.04-x64.Microsoft.DotNet.ILCompiler.Development.nuspec

```diff
@@ -32,6 +32,8 @@
       <file src="../ToolRuntime/*.dll" />
       <file src="../ToolRuntime/*.so" />
       <file src="../ToolRuntime/corerun" target="corerun" />
+      <file src="../../../../packages/toolchain.ubuntu.14.04-x64.Microsoft.DotNet.RyuJit/1.0.0-prerelease/runtimes/ubuntu.14.04-x64/native/libryujit.so" target="ryujit.so" />
```

Comment:
Having to update a million places when we bump up the version of any of these will be a real pain.